src/extension/extension.ts

```diff
@@ -406,7 +406,7 @@ function onTaskProcessEnd(event: vscode.TaskProcessEndEvent) {
       const timeInSeconds = measurePerformance(bazelTaskInfo.startTime);
       vscode.window.showInformationMessage(
         `Bazel ${
-          bazelTaskInfo.command
```

Comment:
I wonder why this changed. Different versions of prettier or tslint?

Reply:
no clue.  fixed.